pkg/s3/bucket.go

```diff
@@ -124,3 +128,38 @@ func SetBucketLifecycle(s3Client *s3.S3, bucketName string) error {
 
 	return err
 }
+func CreateBucketTaggingInput(bucketname string, backUpLocation string) *s3.PutBucketTaggingInput {
```

Comment:
Should the bucket tag key be an input here too, so that this can be resused?

Reply:
Started doing this, getting messier than I'd like. Would prefer to actually do this is a second separate PR.